zproject/urls.py

```diff
@@ -258,6 +258,10 @@
     url(r'^users/me/avatar$', rest_dispatch,
         {'POST': 'zerver.views.user_settings.set_avatar_backend',
          'DELETE': 'zerver.views.user_settings.delete_avatar_backend'}),
+    url(r'^users/me/export-my-data$', rest_dispatch,
+        # FIXME: Use a POST?
+        {'GET': 'zerver.views.user_settings.export_my_data'}),
```

Comment:
Yes, this should be a POST; we don't want browsers to auto-retry this sort of request.